Restructure tasks under node operators (#1197)
smartcontractkit/documentation: src/pages/chainlink-nodes/oracle-jobs/tasks.md

Tasks replace the [core adapters](/chainlink-nodes/oracle-jobs/v1/adapters/) from v1 jobs.
:::

Tasks are a replacement for core adapters that is more flexible. Tasks can be composed in arbitrary order into [pipelines](#writing-pipelines). Pipelines consist of one or more threads of execution where tasks are executed in a well-defined order.

> Chainlink has a number of built-in tasks which are listed below. You can also create your own [external adapters](/chainlink-nodes/external-adapters/external-adapters/) for tasks which are accessed through a `bridge`.

Reviewer: ```suggestion
You can use Chainlink's [built-in tasks](/chainlink-nodes/oracle-jobs/all-tasks), or you can create your own [external adapters](/chainlink-nodes/external-adapters/external-adapters/) for tasks which are accessed through a `bridge`.
```

Reviewer: Editing because the tasks are no longer "listed below" in the same way as before; they're all on another page.
